src/Views/scstormrunoff.html

```diff
@@ -101,9 +101,9 @@ <h4>Citation</h4>
                         </div>
                         <!-- Bohman Urban (1992) tab -->
                         <div ng-if="vm.SelectedTab == 2" style="position:relative;">
-                            <div class="wim-alert hidden-print">Fill out required<span style="color: #ffcc00">*</span> fields. Remaining inputs will be calculated with <b>Calculate Missing Parameters</b> based on delineated basin. Then click <b>Continue</b> to view peak discharge and hydrograph.</div>
+                            <div class="wim-alert hidden-print">Fill out required<span style="color: #ff6700">*</span> fields. Remaining inputs will be calculated with <b>Calculate Missing Parameters</b> based on delineated basin. Then click <b>Continue</b> to view peak discharge and hydrograph.</div>
```

Comment:
Would it make more sense to add class SCorange for elements that use color #ff6700?